src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs

```diff
@@ -11,7 +11,6 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
```

Comment:
@SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-windows`
Error occurred in test script 'D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```